libcloud/compute/drivers/openstack.py

```diff
@@ -2196,20 +2216,26 @@ def _to_volume(self, api_node):
 
         return StorageVolume(
             id=api_node['id'],
-            name=api_node['displayName'],
+            name=api_node.get('name', api_node.get('displayName')),
```

Comment:
wouldn't it make more sense for backwards compatibility to give displayName the priority here? this change would otherwise also affect the OpenStack_1_1_NodeDriver if there is a volume with both a displayName and a name and those names are not the same. I'm not sure if that is ever possible but to be careful I think it would be better to instead do this here: `name=api_node.get('displayName', api_node.get('name')),` What do you think @micafer?

Reply:
I agree, I will change it.